DemoApp/DemoApp/Views/TMBadgesPage.xaml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:modus="clr-namespace:Trimble.Modus.Components;assembly=Trimble.modus.Components"
+             x:Class="DemoApp.Views.TMBadgesPage"
+             Title="TMBadgesPage">
+  <StackLayout Orientation="Vertical" Padding="10" Spacing="10" HorizontalOptions="Fill">
+    <Label Text="Rectangle Badges"/>
+    <StackLayout Orientation="Horizontal" Spacing="10">
+      <modus:TMBadge Text="Text" Color="Primary" Shape="Rectangle" BadgeSize="{Binding BadgeSize}"/>
+      <modus:TMBadge Text="Text" Color="Secondary" Shape="Rectangle" BadgeSize="{Binding BadgeSize}"/>
+      <modus:TMBadge Text="Text" Color="Tertiary" Shape="Rectangle" BadgeSize="{Binding BadgeSize}"/>
+      <modus:TMBadge Text="Text" Color="Success" Shape="Rectangle" BadgeSize="{Binding BadgeSize}"/>
+      <modus:TMBadge Text="Text" Color="Warning" Shape="Rectangle" BadgeSize="{Binding BadgeSize}"/>
+      <modus:TMBadge Text="Text" Color="Error" Shape="Rectangle" BadgeSize="{Binding BadgeSize}"/>
+    </StackLayout>
+    <Label Text="Rounded Badges"/>
+    <StackLayout Orientation="Horizontal" Spacing="10" >
+      <modus:TMBadge Text="Text" Color="Primary" Shape="Round" BadgeSize="{Binding BadgeSize}"/>
+      <modus:TMBadge Text="Text" Color="Secondary" Shape="Round" BadgeSize="{Binding BadgeSize}"/>
+      <modus:TMBadge Text="Text" Color="Tertiary" Shape="Round" BadgeSize="{Binding BadgeSize}"/>
+      <modus:TMBadge Text="Text" Color="Success" Shape="Round" BadgeSize="{Binding BadgeSize}"/>
+      <modus:TMBadge Text="Text" Color="Warning" Shape="Round" BadgeSize="{Binding BadgeSize}"/>
+      <modus:TMBadge Text="Text" Color="Error" Shape="Round" BadgeSize="{Binding BadgeSize}"/>
+    </StackLayout>
+    <modus:TMRadioButtonGroup Orientation="Horizontal" SelectedRadioButtonChangedCommand="{Binding BadgeSizeChangedCommand}">
```

Comment:
This radio button group doesn't have any effect when i ran

Reply:
The initial ChipSize is set to Large , But the initial radio button is set as medium . Now set both to medium and can see a visible change after the first change